.github/actions/git-latest-tag/tests/git-latest-tag.bats

```diff
@@ -0,0 +1,36 @@
+setup() {
+    git fetch --tags
+
+    # Runs everywhere
+    DIR="$( cd "$( dirname "$BATS_TEST_FILENAME" )" >/dev/null 2>&1 && pwd )"
+    PATH="$DIR/..:$PATH"
+
+    # Mock GitHub Actions default variables
+    export GITHUB_ENV=/dev/null
+    export GITHUB_HEAD_REF="AAE-11760-get-latest-tag-sha"
+
+    # Mock git-latest-tag defaults
+    export PATTERN="*"
+    export REPO_DIR="$PWD"
+}
+
+@test "latest_tag" {
+    regex="v[0-9]+\.[0-9]+\.[0-9]+"
+    run git-latest-tag.sh
+    [ "$status" -eq 0 ]
+    if [[ ! $output =~ $regex ]]; then
```

Comment:
I would have checked the entire string via regex but there is also the other test so in the end is fine for me